sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -529,6 +530,9 @@
             <excludePackageNames>
               *.impl*:
               *.implementation*:
+              <!-- RMDocs do utils and samples actually do anything -->
+              *.utils:
+              *.samples
```

Comment:
@srnagar we'd chatted about this earlier. For right now I've opted to leave both of these entries in here. I'm going to add the "RMDocs" as one of the searchable tags that you'll be looking at as part of [Resource Manager library adjustments for azure-client-sdk-parent parentage](https://github.com/Azure/azure-sdk-for-java/issues/13249)